src/mono/cmake/config.h.in

```diff
@@ -1179,6 +1179,9 @@
 /* F_DUPFD_CLOEXEC */
 #cmakedefine HAVE_F_DUPFD_CLOEXEC 1
 
+/* F_FULLFSYNC */
+#cmakedefine HAVE_F_FULLFSYNC 1
```

Comment:
Did you get a build error before adding the change in this file? I'm trying to understand why you would add the mono define as well, because I do not know if the code in pal_io.c is consumed by mono, so I wonder if this change is necessary.

Reply:
I didn't check if this is really required, I just followed the pattern from above flag `HAVE_F_DUPFD_CLOEXEC`. Will remove it and bring it back in case something fails.